setup.py

```diff
@@ -49,7 +49,8 @@ def get_version(package):
 
 
 SCRIPTS = ['iotlab-auth', 'iotlab-experiment', 'iotlab-node', 'iotlab-profile',
-           'iotlab-robot', 'iotlab-admin']
+           'iotlab-robot', 'iotlab-admin',
+           'iotlab']
```

Comment:
could be after iotlab-admin, but on the same line.

Reply:
ok, fixed